portfolio/src/main/webapp/script.js

```diff
@@ -56,4 +56,28 @@ function addBabyPhoto() {
 function removeBabyPhoto() {
     const babyPhotoContainer = document.getElementById('picture-container');
     babyPhotoContainer.innerHTML = '';
-}
\ No newline at end of file
+}
+
+/**
+ * Fetches content from Java Servlet
+ */
+async function fetchContentFromServlet() {
+    const response = await fetch('/data');
+    const arr = await response.json();
+
+    const stringList = document.getElementById('week2-fetch');
+    stringList.innerHTML = '';
+
+    arr.length === 0 ? stringList.innerHTML = '<h3>No Comments Yet</h3>' : stringList.innerHTML = '<h3>Comments:</h3>';
+
+    arr.map(comment => createListElement(comment))
```

Comment:
interesting use of map function :)

Reply:
just trying to practice functional programming (: